core/src/main/kotlin/org/eln2/sim/electrical/mna/Node.kt

```diff
@@ -1,18 +1,34 @@
 package org.eln2.sim.electrical.mna
 
 import org.eln2.debug.dprintln
+import org.eln2.sim.electrical.mna.component.*
+import org.eln2.space.Set
 
+/**
+ * A "node", in MNA; the non-resistive connections between [Component]s.
+ *
+ * Aside from identifying [Component]s' connections, the Nodes' potentials (relative to [Circuit.ground]) are computed as a result of the MNA algorithm.
+ */
 open class Node(var circuit: Circuit) : IDetail {
+	/**
+	 * The potential of this node, in Volts, relative to ground (as defined by the [Circuit]); an output of the simulation.
```

Comment:
Instead of trying to mark it as 'read-only' through documentation, you can make it private and export a public getter.

Reply:
It gets set by `Circuit`, however, which is not in the inheritance chain. Not sure if `friend`s (as in C++) are a thing... :)

Reply:
You want `internal`, then.

Reply:
Throwing an `internal set` on the end of it should be good, then? I'll get to that with the next commit...